cmd/av/commit_amend.go

```diff
@@ -57,6 +61,26 @@ func runAmend(repo *git.Repo) error {
 		commitArgs = append(commitArgs, "--message", commitAmendFlags.Message)
 	}
 
+	tx := db.WriteTx()
+	defer tx.Abort()
+
+	client, err := getGitHubClient()
```

Comment:
I think that making GitHub API call in amend is too much. Can you skip this part and just rely on the local information (e.g. MergeCommit existence)?

Reply:
Whats the concern with calling the API? Too slow? Happy to switch to the local info, but was thinking chances are the local information wont always be up to date so it will not really fix the problem.

Reply:
Slowness and expectation. I think that people would not expect `git commit` to do any network interaction. Keeping the same expectation level for these command would make less surprise. I understand that this wouldn't completely prevent the accidental amend, but that's fine and expected.

Reply:
Have removed this in https://github.com/aviator-co/av/pull/318/commits/d2d97d9630a7200950c7856f130aa9caf95c0522 to unblock this pr, however while I understand the philosophy I think its a worse experience - I wonder if instead it should be configurable. Is there a command that allows you to update the status of all pr's without having to do a full sync of the stacks?